sidesheet/vite.config.ts

```diff
@@ -0,0 +1,22 @@
+import path from "path";
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+
+export default defineConfig({
+  build: {
+    lib: {
+      entry: path.resolve("src", "components", "sidesheet.tsx"),
```

Comment:
Det er vel ikke sidesheet.tsx som bør være entry? Hva med når vi skal eksportere flere ulike komponenter fra dette repoet? Hva med å se på f.eks. cc-components libs shared. Der brukes en index fil for å eksportere alt.

Trenger vi egentlig vite i det hele tatt? Disse pakkene skal være så små som mulig, og jeg tror at vite bare vil gjøre pakkene mye større enn de trenger å være. Kunne vi klart oss uten vite i hele dette repoet? Det er pga vite at yarn.lock er så sykt stor f.eks

Reply:
Spurte om vi skulle bruke vite tidligere og da sa du ja 😇 Men skal fjerne det!

Reply:
Sorry 😢 Jeg tenkte sikkert at det var punch npm pakken det var snakk om eller noe. My bad